### PR TITLE
add docker config

### PR DESCRIPTION
**Pull request by frankobe**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -56,3 +56,5 @@ typings/
 
 # dotenv environment variables file
 .env
+
+data/
```
Comment (dwalintukan) [inline .gitignore]: [gitignore.io](https://gitignore.io) fyi

Comment (frankobe) [reply .gitignore]: @dwalintukan yes. It is a mount vol I use to store db data and qtum data

Comment (dwalintukan) [reply .gitignore]: 👍 ya that website is handy

**Comment by frankobe** [comment]
@dwalintukan @chrisli30 I close this pr because we verify docker config works and this shouldnt be merged into `master` branch. Feel free to open a pr for comparison

**Pull request closed without merging**